modules/tests/shared/src/test/scala/gsp/math/skycalc/package.scala

```diff
@@ -3,22 +3,25 @@
 
 package gsp.math
 
+import coulomb.si._
+import coulomb.refined._
+import eu.timepit.refined.numeric._
 import java.time.ZoneId
 
 package object skycalc {
   val GN =
     Place(
       Lat.fromAngleWithCarry(Angle.fromDoubleDegrees(19.8238068))._1,
       Angle.fromDoubleDegrees(-155.4690550),
-      4213.0,
+      4213.0.withRefinedUnit[NonNegative, Meter],
```

Comment:
It looks like we could remove the `.0` here and in GS. Or `withRefinedUnit` requires a `Double`?

Reply:
Thanks, that's right I'll change it